src/auth.d.ts

```diff
@@ -494,6 +505,18 @@ export namespace admin.auth {
      */
     iss: string;
 
+    /**
+     * The phone number corresponding to the user who the ID token belonged to,
```

Comment:
Suggest "The phone number of the user to whom the ID token belonged, if available."

Or -- can these be present tense "belongs?"  That would be preferable if it is accurate.

Reply:
Done on both.